core/store/migrate/migrations/0259_add_solana_schema.sql

```diff
@@ -3,48 +3,67 @@
 -- This removes evm from the search path, to avoid risk of an unqualified query intended to act on a Solana table accidentally
 -- acting on the corresponding evm table. This makes schema qualification mandatory for all chain-specific queries.
 SET search_path TO public;
```

Comment:
We need to be careful about merging this change if there's a chance it will cause any problems with the evm queries. I think all of the queries are schema-qualified so I think we're okay, but I'm not sure what the original purpose of someone adding evm to the path was. Have you looked into this at all?

Reply:
Yes, we've started using both `evm` and `public` search paths right after creation of evm schema. I assume that it was used as an additional safety measure to avoid errors in queries.
I've also checked manually and it seems that all of the queries are schema-qualified

Reply:
In any case, if a query is still not schema-qualified, it's a bug, and we'll need to address it.